lms/static/certificates/sass/views/_certificate.scss

```diff
@@ -0,0 +1,250 @@
+      padding: ($baseline-v) ($baseline-h);
+    }
+
+    .title {
+      @extend %t-title-6;
+
+      @include media($bp-dl) {
+        @include span-columns(9 of 12);
+      }
+
+      @include media($bp-dm) {
+        @include span-columns(9 of 12);
+      }
+
+      @include media($bp-ds) {
+        @include font-size(14); // can't use %t-title-8 for some reason
+        @include lh(14);
+      }
+
+      @include media($bp-m) {
+        @include font-size(14); // can't use %t-title-8 for some reason
+        @include lh(14);
+      }
+    }
+  }
+
+  // supplemental content
+  .content-supplemental {
+
+    @include media($bp-dm) {
+    }
+
+    @include media($bp-ds) {
+    }
+
+    @include media($bp-m) {
+    }
+
+    .title {
+      @extend %t-title-7;
+      @extend %t-weight3;
+      margin-bottom: ($baseline-v/2);
+    }
+
+    .copy {
+      @extend %t-copy-sub1;
+      color: $gray-l2;
+    }
+
+    .list-actions {
+      margin-top: ($baseline-v);
+
+      .action {
+        @extend %t-copy-sub1;
+        @extend %btn-primary;
+
+        @include media($bp-dm) {
+
+        }
+
+        @include media($bp-ds) {
+          display: block;
+          text-align: center;
+        }
+
+        @include media($bp-m) {
+          display: block;
+          text-align: center;
+        }
+      }
+    }
+  }
+
+  .content-supplemental-how {
+    @extend %trans-size;
+    @include span-columns(3 of 12);
+
+    @include media($bp-dm) {
+      @include span-columns(6 of 12);
+    }
+
+    @include media($bp-ds) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+
+    @include media($bp-m) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+  }
+
+  .content-supplemental-certificates {
+    @extend %trans-size;
+    @include span-columns(3 of 12);
+    @include omega();
+
+    @include media($bp-dm) {
+      @include span-columns(6 of 12);
+    }
+
+    @include media($bp-ds) {
+      @include fill-parent;
+    }
+
+    @include media($bp-m) {
+      @include fill-parent;
+    }
+  }
+
+  .content-supplemental-about {
+    @extend %trans-size;
+    @include span-columns(6 of 12);
+
+    @include media($bp-dm) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+
+    @include media($bp-ds) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+
+    @include media($bp-m) {
+      @include fill-parent;
+      margin-bottom: ($baseline-v*2);
+    }
+  }
+
+  // TODO - Determine what basic types are supported
+  // CASE: honor code
+  &.is-honorcode {
+
+    .status {
+      background: $color-basic-achievement;
+    }
+
+    .certificate-type {
+      color: $color-basic-achievement;
+    }
+  }
+
+  // CASE: verified
+  &.is-idverified {
+
+    .status {
+      background: $color-greater-achievement;
+    }
+
+    .accomplishment {
+      background: $white-t url("../images/logo-idverified-cropped.png") bottom left no-repeat;
```

Comment:
commented above - considerations for openedX watermark + signatures certificate view pushed out to a future story.